core/taiko_genesis/genesis_alloc.go

```diff
@@ -33,3 +33,6 @@ var HeklaGenesisAllocJSON []byte
 
 //go:embed mainnet.json
 var MainnetGenesisAllocJSON []byte
+
+//go:embed preconf_devnet.json
```

Comment:
@cyberhorsey not sure which `taiko-mono` branch I should use for generating the genesis JSON, so I copied the internal devnet genesis JSON here temporarily